test/helpers/application_helper_test.rb

```diff
@@ -80,6 +80,11 @@
       result = markdown("Hey ~~FOO~~ bar")
       result.must_equal "<p>Hey <del>FOO</del> bar</p>\n"
     end
+
+    it "supports nil" do
+      result = markdown("")
```

Comment:
shouldn't that be passing nil for a test?

Reply:
yes 🤦